docbot/action.go

```diff
@@ -13,10 +13,7 @@ import (
 )
 
 const (
-	MessageLabelMissing = `Please provide a correct documentation label for your PR.
-Instructions see [Pulsar Documentation Label Guide](https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0).`
-	MessageLabelMultiple = `Please select only one documentation label for your PR.
-Instructions see [Pulsar Documentation Label Guide](https://docs.google.com/document/d/1Qw7LHQdXWBW9t2-r-A7QdFDBwmZh6ytB4guwMoXHqc0).`
```

Comment:
IIRC `MessageLabelMissing` is referred from the test. I suggest you add a CI task to trigger tests and thus update related code without omitting it. I can do it later today.

Reply:
FYI #72.

Reply:
Thank @tisonkun!